local_planner/cfg/local_planner_node.cfg

```diff
@@ -7,7 +7,7 @@ gen = ParameterGenerator()
 
 # local_planner
 
-gen.add("box_radius_",    double_t,    0, "Minimum box extension in x direction", 7,  0, 20)
+gen.add("box_radius_",    double_t,    0, "Data points farther away will be discarded", 7,  0, 20)
```

Comment:
I said it before and I'll say it again: I don't like the fact that we're violating basic geometry nomenclature.

It's either the side of a cube, the length/width/height of a box or the radius of a sphere...

Reply:
the problem is the "box" as it is implemented is neither of all those shapes. In reality it is a sphere with this radius, all points farther away will be ignored. On top of that we use the distance sensor data to determine, which points could belong to the ground and cut off the sphere from the bottom. So I think the description I provided is way more accurate than it was before, but the long explanation just does not fit. We could argue to change the term "box" in the whole code as it is a sphere rather than a box. But actually I just wanted to change the parameter...